math/geometry/distance.go

```diff
@@ -0,0 +1,36 @@
+// distance.go
+// Find Euclidean distance between two points
+// author(s) [Chetan Patil](https://github.com/Chetan07j)
+
+// Package geometry contains geometric algorithms
+package geometry
+
+import (
+	"errors"
+	"math"
+)
+
+// Point defines a point with x and y coordinates.
```

Comment:
```suggestion
// EuclideanPoint defines a point with x and y coordinates.
```

Whenever refactoring, it is also a good idea to refactor all the comments associated with them. Otherwise the comments and code go out of sync.

Reply:
Done @tjgurwara99